src/main/scala/cz/nomi/locusRflktAddon/display/Pages.scala

```diff
@@ -284,6 +286,8 @@ object Const {
     val elevationDownhillWorkout = "ELEVATION_WORKOUT_DOWNHILL"
     val cadenceCurrent = "BIKE_CAD_CURRENT"
     val heartRateCurrent = "HR_CURRENT"
+    val averageCadenceWorkout = "CADENCE_WORKOUT_AVG"
+    val averageHrmWorkout = "HRM_WORKOUT_AVG_MOVE"
```

Comment:
Why does hrm have _MOVE but cadence not?
(I'd expect that this depends on whether "record only when moving" is checked in Locus. Is that not so?)